lib/matplotlib/axes/_axes.py

```diff
@@ -4465,21 +4465,25 @@ def stackplot(self, x, *args, **kwargs):
                          label_namer=None)
     def streamplot(self, x, y, u, v, density=1, linewidth=None, color=None,
                    cmap=None, norm=None, arrowsize=1, arrowstyle='-|>',
-                   minlength=0.1, transform=None, zorder=2, start_points=None):
+                   minlength=0.1, maxlength=4.0, transform=None, zorder=2,
```

Comment:
All new args need to go at the end to maintain back-compatibility.  The reason is if people are (un wisely!) passing in everything as positional args.

When we can drop python2 _and_ can break API again most of these should be converted to kwarg only, but for now they need to go at the end.